torch/_inductor/triton_heuristics.py

```diff
@@ -170,6 +170,8 @@ def __init__(
         # pre-create the profiler context manager to reduce latency
         self.record_function_ctx = torch._C._profiler._RecordFunctionFast(
             self.meta.get("kernel_name", "triton kernel")
+            + ", OriginOps: "
+            + str(self.meta.get("origin_ops", []))
```

Comment:
ideally we want these to go in the metadata instead of appearing directly in the name of the event... but for now this is probably fine

Reply:
I was looking into how you could make the origin_ops or any other attribute go into the nvtx markers in a field other than name.  The nvtx range API provides a 'payload' field that you can set with string or numeric values.  The problem is that you need to use a different function than is currently used in pytorch.  The function we want is [nvtxRangePushEx](https://docs.nvidia.com/nsight-visual-studio-edition/nvtx/index.html). The user can pass in an eventAttrib where you can set the payload.  Currently pytorch uses the[ nvtxRangePushA()](https://github.com/pytorch/pytorch/blob/main/torch/csrc/profiler/stubs/cuda.cpp#L78) function which only accepts a string message as an argument.
For Kineto, I think the [RecordFunction::before() ](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/record_function.h#L289-L309)method would need to be modified in order to support an extra string type argument to carry the origin_ops or other optional attributes.